modules/log-collector-resources-scheduling.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * logging/log_collection_forwarding/log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log-collector-resources-scheduling_{context}"]
+= Configuring resources and scheduling for logging collectors
+
+Administrators can modify the resources or scheduling of the collector by creating a `ClusterLogging` custom resource (CR) that is in the same namespace and has the same name as the `ClusterLogForwarder` CR that it supports.
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {clo} version 5.8 or newer.
+* You have created a `ClusterLogForwarder` CR.
+* The multi log forwarding feature is enabled on your cluster.
```

Comment:
Consider striking this bullet all-together.

* There is no real action to "enable" mclf
* These statements as written apply whether it is the "legacy" deployment or mclf deployment